cmd/vic-machine/create/create.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/vmware/vic/lib/install/data"
 	"github.com/vmware/vic/lib/install/management"
 	"github.com/vmware/vic/lib/install/validate"
+
```

Comment:
nit: I don't think this line should be here

Reply:
Agreed, fixed